examples/Sqlpp11/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2016-2017, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-sqlpp11)
+
+hunter_add_package(Sqlpp11)

```

Comment:
> SphinxWarning: /home/travis/build/ruslo/hunter/docs/packages/pkg/Sqlpp11.rst:18:start-after pattern not found: # DOCUMENTATION_START {
Warning, treated as error:
/home/travis/build/ruslo/hunter/docs/packages/pkg/Sqlpp11.rst:18:start-after pattern not found: # DOCUMENTATION_START {
The command "./jenkins.sh" exited with 1.

`DOCUMENTATION_START {` / `DOCUMENTATION_END }` are missing. Please check template file:
- https://github.com/ruslo/hunter/blob/v0.23.133/examples/foo/CMakeLists.txt#L12-L18

Reply:
Oops. Didn't know that's where it gets it from.
Thanks.

Reply:
Yes, just copy it:
```
[hunter]> cp docs/packages/pkg/foo.rst docs/packages/pkg/hunter_box_1.rst
[hunter]> sed -i 's,foo,hunter_box_1,g' docs/packages/pkg/hunter_box_1.rst
```
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation